### ISSUE TITLE
we need to fix fastqc reporting

### ISSUE THREAD

**Issue by ctb** [ngs-docs/angus#178]
apt-get install doesn't work properly for fastqc - see https://angus.readthedocs.io/en/2017/quality-trimming.html.

I think we can just install the prereqs and then download fastqc / unpack it ourselves. but worth trying out.

**Comment by upendrak** [comment]
I just ran `sudo apt-get -y install fastqc` on the Ubuntu 14.04 base image and it installed the fastqc just fine..

**Comment by ctb** [comment]
right, but we're using 16.04 for everything else! and at least for some newer
software 14.04 is too painful and slow (involves recompiling many recent Python
packages, for example).

**Comment by bluegenes** [comment]
the jetstream boot tutorial currently has 14.04 - I'll update the images (login-9.png /thumb.png are the only changes) + text with 16.04

**Comment by upendrak** [comment]
I tested most of the tutorials with 14.04 and that's why I was confused. Anyway my 16.04 has just started deploying and it is taking longer time compared to 14.04. I just hope this is one of case..

**Comment by ctb** [comment]
good catch, thx and sorry!

**Comment by bluegenes** [comment]
boot tutorial has m1.medium -- will that be ok, or should I change to m1.small?

**Comment by upendrak** [comment]
I was told that we will be able to launch m1.medium for all the students. So let's keep it as it is for now.

**Comment by bluegenes** [comment]
great, thx!

**Comment by upendrak** [comment]
I found a way to update the fastqc installed using `apt-get install fastqc` on Ubuntu 16.04.

```
wget https://launchpad.net/ubuntu/+archive/primary/+files/fastqc_0.11.5+dfsg-3_all.deb
sudo dpkg -i fastqc_0.11.5+dfsg-3_all.deb
sudo apt-get install -f
```

After doing this, `fastqc` commands works just fine. Shall I do a PR @ctb

**Comment by ctb** [comment]
yes please thank you!!

On Thu, Jun 22, 2017 at 11:39:51AM -0700, Upendra Kumar Devisetty wrote:
> I found a way to update the fastqc installed using `apt-get install fastqc` on Ubuntu 16.04.
> 
> ```
> wget https://launchpad.net/ubuntu/+archive/primary/+files/fastqc_0.11.5+dfsg-3_all.deb
> sudo dpkg -i fastqc_0.11.5+dfsg-3_all.deb
> sudo apt-get install -f
> ```
> 
> After doing this, `fastqc` commands works just fine. Shall I do a PR @ctb 
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/ngs-docs/angus/issues/178#issuecomment-310466745
-- 
C. Titus Brown, ctbrown@ucdavis.edu

**Comment by ctb** [comment]
fixed in #184.
